src/benchmarks/micro/corefx/System.Buffers/ReadOnlySequenceTests.cs

```diff
@@ -28,27 +29,73 @@ public class ReadOnlySequenceTests<T>
         [Benchmark(OperationsPerInvoke = 16)]
         public int FirstArray() => First(new ReadOnlySequence<T>(_array));
 
+#if NETCOREAPP3_0
```

Comment:
@billwert, @jorive, I am just curious, is this the expected way to single out API benchmarks that only exist on certain TFMs? What if I want to add benchmarks for types that only exist on netstandard2.0, or only exist on .NET Core 2.0+, etc.?

Reply:
good question. I don't think we have a solid plan yet. (And this approach is fragile, of course.) @adamsitnik may have considered this - let's see what he says when he's back from vacation.

Reply:
@ahsonkhan it depends. if it's a single method or just a few like here, you can use `#if TFM`. If it's an entire type, you should exclude it per certain TFM in the `.csproj` file

https://github.com/dotnet/performance/blob/eaec460c81e07410bb293b68d77315fe90ecd2f9/src/benchmarks/micro/MicroBenchmarks.csproj#L60-L61

Reply:
Sure, I figured that would be the expected approach. That's what I ended up doing for the recent JsonWriter perf tests. Thanks for clarifying.